### PR TITLE
fix for changes in dolphin

### PR DESCRIPTION
**Pull request by mirzaees**
Minor fix for some changes in dolphin

### CODE REVIEW

File: src/disp_s1/product.py [hunk 1]
```diff
@@ -559,54 +559,57 @@ def _create_grid_mapping(group, crs: pyproj.CRS, gt: list[float]) -> h5netcdf.Va
...
-def create_compressed_products(comp_slc_dict: dict[str, Path], output_dir: Filename):
+def create_compressed_products(
+    comp_slc_dict: dict[str, list[Path]], output_dir: Filename
+):
     """Make the compressed SLC output product."""
 
     def form_name(filename: Path, burst: str):
         # filename: compressed_20180222_20180716.tif
-        date_str = _format_date_pair(*get_dates(filename.stem))
+        date_str = format_date_pair(*get_dates(filename.stem, fmt="%Y%m%d"))
```
Comment (scottstanie) [inline src/disp_s1/product.py:569]: Do we want this to be `format_dates(*get_dates...)` now that there's 3?

**Review by scottstanie — APPROVED** [review]

File: src/disp_s1/product.py (hunk 1 above)
Comment (mirzaees) [reply src/disp_s1/product.py:569]: yes, thank you

**Comment by scottstanie** [comment]
FYI it's failing because i changed the GHA workflow to just test the `main` branch of dolphin, and a release (set at 0.12 now). so we'll have to do a new release first, but it looks okay

File: src/disp_s1/main.py [hunk 2]
```diff
@@ -1,9 +1,10 @@
 from __future__ import annotations
 
-from dolphin import get_dates, utils
+from dolphin import utils
 from dolphin._log import get_log, log_runtime
 from dolphin.workflows.config import DisplacementWorkflow
 from dolphin.workflows.displacement import run as run_displacement
+from opera_utils._dates import get_dates
```
Comment (scottstanie) [inline src/disp_s1/main.py:7]:
```suggestion
from opera_utils import get_dates
```

**Comment by scottstanie** [comment]
with https://github.com/isce-framework/dolphin/pull/228 this should now be working and ready

**Pull request merged**